packages/trpc/server/routers/viewer/organizations/getBrand.handler.ts

```diff
@@ -0,0 +1,30 @@
+import { prisma } from "@calcom/prisma";
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+import type { TGetBrandSchema } from "./getBrand.schema";
+
+type VerifyCodeOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TGetBrandSchema;
+};
+
+export const getBrandHandler = async ({ ctx, input }: VerifyCodeOptions) => {
+  const { orgId } = input;
```

Comment:
Should we just get take `orgId` from ctx.user? I saw this in one of @sean-brydon's PR, then we don't need any input

Reply:
Good idea, I was actually passing the org from session and then checking it was the same value, may as well grab it directly from the context user. Thanks!